bitbots_localization/include/bitbots_localization/localization.h

```diff
@@ -219,8 +219,8 @@ class Localization {
 
   void getMotion();
 
-  geometry_msgs::Vector3 movement_;
-  geometry_msgs::Vector3 movement2_;
+  geometry_msgs::Vector3 cartesian_movement_;
```

Comment:
you call it linear movement in the motion model. I like that better than cartesian movement.